src/main/java/fr/delthas/skype/WebConnector.java

```diff
@@ -146,9 +152,13 @@ private User updateUser(JSONObject userJSON, boolean newContactType) throws Pars
         userUsername = mri.substring(senderBegin + 1);
         userDisplayName = userJSON.optString("display_name", null);
         JSONObject profileJSON = userJSON.getJSONObject("profile");
-        JSONObject nameJSON = profileJSON.getJSONObject("name");
-        userFirstName = nameJSON.optString("first", null);
-        userLastName = nameJSON.optString("surname", null);
+        
```

Comment:
I removed the default set-s for firstName and lastName, since when not set we should return null and let the library user decide what to do, rather than return a default value ourselves.